test/error-messages/nested-invalid-value.txt

```diff
@@ -0,0 +1,9 @@
+Invalid value
+
+concat :: Array a -> Array a -> Array a
+                     ^^^^^^^
+                        1
+
+1) "ab" :: String
+
+The value at position 1 is not a member of (Array Number).
```

Comment:
:zap:ed all the rest, will look into this one.

Also I went back to including a blank line between the position detail lines.. it feels like an extravagant amount of white space so I want to like it better without, but I don't :sheep: